include/pybind11/buffer_info.h

```diff
@@ -53,7 +53,11 @@ struct buffer_info {
 
     explicit buffer_info(Py_buffer *view, bool ownview = true)
     : buffer_info(view->buf, view->itemsize, view->format, view->ndim,
-            {view->shape, view->shape + view->ndim}, {view->strides, view->strides + view->ndim}, view->readonly) {
+	    {view->shape, view->shape + view->ndim},
+            view->strides
```

Comment:
Please also add a comment here, saying that we need to do this because `ctypes` seem to be misbehaving.

Reply:
Good idea.